packages/beacon-node/src/api/impl/beacon/state/index.ts

```diff
@@ -252,8 +252,31 @@ export function getBeaconStateApi({
       }
 
       const epoch = filters.epoch ?? computeEpochAtSlot(state.slot);
+      const currentEpoch = computeEpochAtSlot(state.slot);
```

Comment:
can potentially avoid call to `computeEpochAtSlot` here

```suggestion
      const stateEpoch = computeEpochAtSlot(state.slot);
      const epoch = filters.epoch ?? stateEpoch;
```

also this is the epoch of the state, not the current epoch (based on clock)